src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const seenValues = {};
+  const result = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (seenValues[value]) {
```

Comment:
you don't need a separate `seenValues` object
if this value was seen before than the `result` object has such key